test/Templates.Test/CdnScriptTagTests.cs

```diff
@@ -130,9 +176,10 @@ private static string GetFileContentFromArchive(ScriptTag scriptTag, string rela
             return null;
         }
 
-        private static List<ScriptTag> GetScriptTags(string zipFile)
+        private static (List<ScriptTag> Scripts, List<LinkTag> Links) GetScriptTags(string zipFile)
```

Comment:
Change the method name?

Reply:
nit: tuple members are defined in lowercase